src/app/page.tsx

```diff
@@ -5,14 +5,14 @@ import { Logo } from '@/components'
 
 export default function Home() {
   return (
-    <>
+    <main className="stars-layout">
```

Comment:
Maybe stars layout should be a component rather than CSS that I have to remember.